.github/workflows/mypy.yaml

```diff
@@ -0,0 +1,50 @@
+name: "mypy"
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches:
+      - develop
+
+concurrency:
+
+  group: "${{ github.ref }}-${{ github.head_ref }}-${{ github.workflow }}"
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  mypy :
+    if: "github.repository == 'MDAnalysis/mdanalysis'"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: setup_miniconda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: 3.8
+          auto-update-conda: true
+          channel-priority: flexible
+          channels: conda-forge
+          add-pip-as-python-dependency: true
+          architecture: x64
```

Comment:
Looks like you need `mamba-version: "*"` for mamba to be installed. See https://github.com/MDAnalysis/mdanalysis/blob/ec5da1de682322c436970252f03b5a24ea673113/.github/workflows/gh-ci.yaml#L88

Reply:
yes  mentor , I just realized after making the commit .

Reply:
I'll wait a bit before giving away too many answers, then :wink:

Reply:
Actually am getting the error , nothing provides numpy 1.20.0 . What am I supposed to do about it ?

Reply:
The problem is the way you specify the numpy version. It is a bit unintuitive: the `numpy` attribute expects `numpy=1.20.0` and not just `1.20.0`.

Since, we are at it, you should also check the `channel` attribute.

Reply:
Oh, you already checked `channel` :smile:

Reply:
I think the workflow runs properly now 😄.